presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -4553,6 +4553,38 @@ public void testRowNumberMultipleFilters()
         assertEqualsIgnoreOrder(actual.getMaterializedRows(), expected.getMaterializedRows());
     }
 
+    @Test
+    public void testRowNumberInvalidFilters()
```

Comment:
"Invalid" is not the right characterization. The queries are perfectly valid. They are a no-op, though.